src/app.js

```diff
@@ -52,18 +70,60 @@ class MyFHIRServer extends FHIRServer.Server {
 
         return this;
     }
+
+    configureHtmlRenderer() {
+        if (isTrue(env.RENDER_HTML)) {
+            this.app.use((req, res, next) => {
+                // console.log('--- user agent ----');
+                // console.log(req.useragent);
+                const parts = req.url.split(/[/?,&]+/);
+                console.log('----- parts -----');
+                console.log(parts);
+                if (parts && parts.length > 1 && !parts.includes('raw=1')) {
+                    const resourceName = parts[2].toLowerCase();
```

Comment:
Would this cause an index out of bounds error since we are checking if its greater than 1?

Reply:
good catch.  fixed it.